cmd/destroy/all.go

```diff
@@ -28,7 +28,7 @@ type localDestroyAllCommand struct {
 }
 
 func newLocalDestroyerAll(
-	k8sClient *kubernetes.Clientset,
+	k8sClient kubernetes.Interface,
```

Comment:
Why do we need this we can create it with the provider. Injecting it can cause throttling problems, that's why we should always use the provider